src/activitylog/activitylog.js

```diff
@@ -1 +1,5 @@
-/* activitylog.js */
+import ActivityLog from '../lib/ext-activitylog.js';
+
+window.addEventListener('DOMContentLoaded', async () => {
```

Comment:
In most cases may not make any actualy difference, nevertheless technically the target of the DOMContentLoaded event should have the `document` as its actual target and so it may be reasonable to add the listener for this event directly on the `document` element, even if the event listener can be registered on both the `window` and the `document`.

Reply:
Okay, using `document` then :-)